.travis.yml

```diff
@@ -1,4 +1,5 @@
 language: python
+python: 3.5
```

Comment:
This line isn't needed. Other than that it LGTM. Remove it and I'll merge. :) Thanks in advance for the contribution.